TEMPLATE.md

```diff
@@ -41,7 +41,7 @@ Express here the potential negative ramifications of taking on this proposal. Wh
 
 Think through the evolution of this proposal well into the future. How do you see this playing out? What would this proposal result in in one year? In five years?
 
-## Suggested implementation timeline
+## Suggested deployment timeline
```

Comment:
Let's make it "Suggested implementation and deployment timeline" instead? So far many AIPs only drafted when the code is almost complete, but for future AIPs, we would want to know when the code will be implemented and land on main branch too.

Reply:
Okay. I added two separate sections: one on implementation & one on deployment, just so everything is organized neatly.

Reply:
If we'd like to know when the code "lands", then I can also update the deployment timeline instructions to include an estimate of when they think the PR should land by.